Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -148,27 +156,53 @@ export type NotificationProps = {
 };
 
 const makeLinkLabel = (link: NotificationLink) => (
-  <ActionButton css={linkButton} onClick={link.onClick}>
+  <ActionButton styles={linkButton} onClick={link.onClick}>
     {link.label}
   </ActionButton>
 );
 
 const defaultCardContentRenderer = (props: CardProps) => {
-  const { title, description, type, link, links } = props;
+  const { title, description, type, link, links, stretchLinks } = props;
+
+  const linkList = links ?? [link ?? null];
+
+  const stackProps: IStackProps = {
+    horizontalAlign: stretchLinks ? 'space-between' : 'end',
+    tokens: {
+      childrenGap: stretchLinks ? undefined : '20px',
+      padding: '0 16px 0 0',
+      maxHeight: '24px',
+    },
+  };
+
   return (
     <div css={cardContent}>
       {type === 'error' && <Icon css={errorType} iconName="ErrorBadge" />}
       {type === 'success' && <Icon css={successType} iconName="Completed" />}
       {type === 'warning' && <Icon css={warningType} iconName="Warning" />}
       {type === 'question' && <Icon css={questionType} iconName="UnknownSolid" />}
       {type === 'congratulation' && <Icon css={congratulationType} iconName="Trophy2Solid" />}
+      {type === 'custom' && (
+        <Icon
+          css={css`
+            margin-top: ${iconMargin};
+            color: ${props.color ?? SharedColors.gray10};
+          `}
+          iconName={props.icon ?? 'UnknownSolid'}
+        />
+      )}
       <div css={cardDetail}>
         <div css={cardTitle}>{title}</div>
         {description && <div css={cardDescription}>{description}</div>}
-        {link && makeLinkLabel(link)}
-        {links?.map((link) => (
-          <div key={link.label}>{makeLinkLabel(link)}</div>
-        ))}
+        <Stack horizontal {...stackProps}>
```

Comment:
Can't you just use the `horizontal-align` prop  instead of adding spacers? 
```suggestion
        <Stack horizontal horizontal-align="space-between" {...stackProps}>
```

Reply:
That would align things evenly; the design calls for extra space between two of the options.